packages/scope-manager/package.json

```diff
@@ -27,17 +27,6 @@
   "license": "MIT",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "scripts": {
-    "build": "tsc -b tsconfig.build.json",
-    "postbuild": "downlevel-dts dist _ts3.4/dist",
-    "clean": "tsc -b tsconfig.build.json --clean",
-    "postclean": "rimraf dist && rimraf _ts3.4 && rimraf coverage",
-    "format": "prettier --write \"./**/*.{ts,js,json,md}\" --ignore-path ../../.prettierignore",
-    "generate:lib": "../../node_modules/.bin/ts-node --files --transpile-only tools/generate-lib.ts",
-    "lint": "eslint . --ext .js,.ts --ignore-path='../../.eslintignore'",
-    "test": "jest --coverage",
-    "typecheck": "tsc -p tsconfig.json --noEmit"
```

Comment:
Is there any way we can leave these in so that we can still run `yarn build` et al?
I personally use these quick shortcuts all the time when working in packages.

We could just make them aliases to the `nx` commands.

This also makes it easier for contributors as it means they don't need to specifically know about nx or that we use it.